pom.xml

```diff
@@ -66,19 +68,21 @@
       explicitly specifying the latest version here because one we get from the parent POM
       tends to lag behind a bit
     -->
-    <maven-hpi-plugin.version>1.95</maven-hpi-plugin.version>
+    <maven-hpi-plugin.version>1.106</maven-hpi-plugin.version>
+    <mesos.version>0.21.1</mesos.version>
+    <protobuf.version>2.5.0</protobuf.version>
   </properties>
 
   <dependencies>
       <dependency>
           <groupId>org.apache.mesos</groupId>
           <artifactId>mesos</artifactId>
-          <version>0.20.0</version>
+          <version>${mesos.version}</version>
```

Comment:
It does allow you to trial building the plugin with a new version of mesos without physically changing the pom, for example when a new version does came out and you want to trial building you could do the following:
    mvn clean package -Dmesos.version=0.22

With regards to generally changing these values to variables, that has simply been a standard I have tended to follow in pom files, I find it provides a nice concise way to see upfront (in the properties section) what all the main dependencies and versions are being used - however if you feel this is not beneficial, then I am happy to remove it as well, just let me know.